src/OpenSolid/BoundingBox2d.elm

```diff
@@ -530,14 +531,16 @@ overlappingBy order tolerance =
 
 
 {-| Check if one box is separated from another by less than, greater than or
-equal to a given amount. If you were performing clash detection between some
-objects, you could use `separatedBy` on those objects' bounding boxes as a quick
-check to see if the objects had a gap of at least 1 cm between them:
+equal to a given amount. For example, if you were performing clash detection
```

Comment:
I think it would read better at present tense: "if you are performing ..., you can use ...". Or at least, it does not feel natural for me.

Reply:
Just pushed a slightly modified version ("For example, to perform clash detection between some objects, you could..."). Hopefully that reads a bit better!